projects/storefrontstyles/scss/components/content/navigation/_navigation-ui.scss

```diff
@@ -35,6 +35,7 @@
       > nav > .wrapper {
         padding: 15px 20px;
         background: #fff;
+        box-shadow: rgba(0, 0, 0, 0.15) 0px 7px 15px 0px;
```

Comment:
I took a look at the mockups and i found `rgb(0 0 0 0.15) 0px 3px 12px 0px`. I think this is the correct box shadow